### PR TITLE
Task11 best practices

### PR DESCRIPTION
**Pull request by bandax**
[lighthouse report react second.pdf](https://github.com/bandax/Movies.ReactJS/files/5535255/lighthouse.report.react.second.pdf)

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+build/*
+public/*
+types/*
+src/__tests__/*
+node_modules
+coverage
+.eslint.js
+src/stories/*
+src/server/*
+__mocks__
```
Comment (endiny) [inline .eslintignore:11]: empty line

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (endiny) [inline .storybook/preview.js:1]: remove empty line

File: .storybook/preview.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+}
```
Comment (endiny) [inline .storybook/preview.js:4]: add empty line

File: src/stories/Introduction.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,207 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```
Comment (endiny) [inline src/stories/Introduction.stories.mdx:1]: Do we need that file?

File: src/stories/Introduction.stories.mdx_p [hunk 5]
```diff
@@ -0,0 +1,216 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```
Comment (endiny) [inline src/stories/Introduction.stories.mdx_p:1]: and this?

**Review by endiny — CHANGES_REQUESTED** [review]

File: .eslintignore (hunk 1 above)
Comment (bandax) [reply .eslintignore]: done

File: .storybook/preview.js (hunk 2 above)
Comment (bandax) [reply .storybook/preview.js]: done

File: .storybook/preview.js (hunk 3 above)
Comment (bandax) [reply .storybook/preview.js]: done

File: src/stories/Introduction.stories.mdx (hunk 4 above)
Comment (bandax) [reply src/stories/Introduction.stories.mdx:1]: yes, part of the introduction to story book components

File: src/stories/Introduction.stories.mdx_p (hunk 5 above)
Comment (bandax) [reply src/stories/Introduction.stories.mdx_p:1]: yes, part of the introduction to story book components

**Pull request merged**